### ISSUE TITLE
Implementation of Ao evaluation index of got10k val dataset

### ISSUE THREAD

**Issue by davidyang180** [researchmm/Stark#45]
Hello!About the implementation of Ao evaluation index of got10k val dataset, which code file is it implemented in?

**Comment by l-sf** [comment]
Hello, have you found how to calculate the result AO? If you have find it, please tell me, thank you very much!!

**Comment by davidyang180** [comment]
> Hello, have you found how to calculate the result AO? If you have find it, please tell me, thank you very much!!

Hi! The performance evaluation of got10k are generally to submit the test results on the got official website after training with the training set.

**Comment by l-sf** [comment]
> 
I know this, but the official website is evaluating the performance of the test set. I want to know how to evaluate the performance of the val set, because the official website has a limit on the number of submissions.

**Comment by davidyang180** [comment]
> > 
> 
> I know this, but the official website is evaluating the performance of the test set. I want to know how to evaluate the performance of the val set, because the official website has a limit on the number of submissions.

Don't worry, after you submit, you will find that the got official website can be submitted indefinitely for evaluation.
